src/Docker/Manager/ImageManager.php

```diff
@@ -104,15 +104,22 @@ public function find($repository, $tag = 'latest')
      * @throws \Docker\Exception\UnexpectedStatusCodeException
      * @throws \GuzzleHttp\Exception\RequestException
      *
-     * @return ImageManager
+     * @return @return json data from docker inspect
      */
     public function inspect(Image $image)
     {
         try {
-            $response = $this->client->get(['/images/{id}/json', ['id' => $image->__toString()]]);
+            # Prefer the unique hash id for searching
```

Comment:
If you really want to add a comment here, I'd rather have a real explanation like the one you gave in the PR description :)